src/main/java/com/muxiu1997/sharewhereiam/event/EventHandler.java

```diff
@@ -0,0 +1,71 @@
+package com.muxiu1997.sharewhereiam.event;
+
+import static com.muxiu1997.sharewhereiam.network.NetworkHandler.network;
+import static com.muxiu1997.sharewhereiam.util.WaypointUtil.waypointOfLocation;
+import static com.muxiu1997.sharewhereiam.util.WaypointUtil.waypointOfRayTrace;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraftforge.client.event.RenderWorldLastEvent;
+import net.minecraftforge.event.entity.EntityJoinWorldEvent;
+
+import org.lwjgl.input.Keyboard;
+
+import com.muxiu1997.sharewhereiam.client.key.KeyShare;
+import com.muxiu1997.sharewhereiam.integration.Mods;
+import com.muxiu1997.sharewhereiam.integration.journeymap.WaypointManager;
+import com.muxiu1997.sharewhereiam.integration.journeymap.WaypointMarker;
+import com.muxiu1997.sharewhereiam.network.MessageMarkWaypoint;
+import com.muxiu1997.sharewhereiam.network.MessageShareWaypoint;
+import com.muxiu1997.sharewhereiam.util.WaypointUtil;
+
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import cpw.mods.fml.common.gameevent.InputEvent;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+
+public class EventHandler {
+
+    public static final EventHandler INSTANCE = new EventHandler();
```

Comment:
Similar to commands, I don't see any value in storing this `INSTANCE` variable over just calling `new EventHandler()` on registration